models/intermediate/int_google_play__earnings.sql

```diff
@@ -0,0 +1,30 @@
+{{ config(enabled=var('google_play__using_earnings', True)) }} -- maybe this should be disabled by default? 
+
+with earnings as (
+
+    select *
+    from {{ var('earnings') }}
+
+), daily_country_metrics as (
+
+-- let's pivot out revenue metrics associated wit each type of transaction type
+{% set transaction_types = dbt_utils.get_column_values(table=ref('stg_google_play__earnings'), column="coalesce(transaction_type, 'other')") %}
+
+    select 
+        transaction_date as date_day,
+        buyer_country as country, -- mm should we include states here as well? we'd have to roll up to country to join with subscriptions
```

Comment:
hm yeah i wonder if it would be best to create a state-level daily model in the combo package itself (if the main reason to do so would be to combine with the Apple Store at the same grain, perhaps it shouldn't be in the google-play-only package) or just add it here and still roll up to the country-grain in the finance report (since subscriptions are only at the country level)

Reply:
Ah I didn't realize subscriptions were only at the country level. If that is the case then I don't see how state would provide much value to the google play only package. Let me know if you disagree. Then we can handle any aggregation necessary operations within the roll up package to allow for them to be unioned.